src/Logger/Log.php

```
@@ -312,7 +312,7 @@ public function addExtra(string $key, $value): void
      *
      * @return array
      */
-    public function getExtra(): array
+    public function getExtra(): array // @phpstan-ignore-line
```

please add a type rather than ignore the error.